codegen/package.go

```diff
@@ -197,15 +198,112 @@ func (p PackageHelper) TypeFullName(typeSpec compile.TypeSpec) (string, error) {
 	if typeSpec == nil {
 		return "", nil
 	}
-	tfile := typeSpec.ThriftFile()
+	return goType(p, typeSpec)
+}
+
+// goType returns the Go type string representation for the given thrift type.
+func goType(p PackageHelper, spec compile.TypeSpec) (string, error) {
```

Comment:
This function is duplicate with https://github.com/uber/zanzibar/blob/master/codegen/type_converter.go#L68